docker/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN rustup self update \
   && rustup show \
   && rustup component add rustfmt
 
-
+RUN cargo install flamegraph
```

Comment:
Ideally we should prefer install binary directly, but it seems not very slow to compile, so acceptable to me.